jobs/commons/src/utils/tests.utils.ts

```diff
@@ -9,10 +9,8 @@ import { randomUUID } from 'crypto'
 type RecursivePartial<T> = {
   [P in keyof T]?: RecursivePartial<T[P]>
 }
-function createMockFactory<TDefaultValue>(
-  defaultValue: TDefaultValue
-): <T>(overwrites?: RecursivePartial<TDefaultValue>) => T {
-  return <T>(overwrites: RecursivePartial<TDefaultValue> = {}) => {
+function createMockFactory<T>(defaultValue: T): (overwrites?: RecursivePartial<T>) => T {
+  return (overwrites: RecursivePartial<T> = {}): T => {
```

Comment:
The returned function does not accept generics anymore, it takes the data type from his factory function